app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/upcoming/MealBottomSheet.kt

```diff
@@ -0,0 +1,224 @@
+package com.cornellappdev.android.eateryblue.ui.components.upcoming
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.android.eateryblue.R
+import com.cornellappdev.android.eateryblue.ui.components.general.Filter
+import com.cornellappdev.android.eateryblue.ui.theme.EateryBlue
+import com.cornellappdev.android.eateryblue.ui.theme.EateryBlueTypography
+import com.cornellappdev.android.eateryblue.ui.theme.GrayZero
+
+@Composable
+fun MealBottomSheet(
+    selectedFilters: MutableList<Filter>,
+    hide: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .padding(start = 16.dp, end = 16.dp, top = 24.dp)
```

Comment:
you can use horizontal = 16.dp instead of just start and end. Makes the code slightly more clean